src/api/json-api.ts

```diff
@@ -0,0 +1,23 @@
+export interface Relation {
+  link?: string;
+  data: APIModel | null;
+}
+
+export interface CreateRelation {
+  data: APIModel | null;
+}
+
+export interface Relations {
```

Comment:
Maybe something like `oneToManyRelation`?
It's longer, but I find this distinction a bit enigmatic ;)